SJMaster/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 from django.urls import path, include
 
+
```

Comment:
I think you can remove this new line no need to touch this file in this PR

Reply:
ok